### PR TITLE
updated project documents: code of conduct, contributor and contributing

### LINKED ISSUE #3: Task: Editing the Project Documents: contributor.md, conduct.md, contributing.md
Edit the project documents tailor to our project

### PR DESCRIPTION
**Pull request by ChuckHo777**
Adapting the default files and edit the contact and contributors info.

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -59,12 +59,13 @@ Get Started!
 
 Ready to contribute? Here's how to set up `simpler_eda` for local development.
```
Comment (nichowu) [inline CONTRIBUTING.rst]: Ready to contribute? 

In general, we ask you to follow the [GitHub Flow workflow](https://guides.github.com/introduction/flow/). 

Here's how to set up `simpler_eda` for local development.

**Review by nichowu — APPROVED** [review]
approved! LGTM!

File: CONTRIBUTING.rst (hunk 1 above)
Comment (ChuckHo777) [reply CONTRIBUTING.rst]: Got it!

Comment (ChuckHo777) [inline CONTRIBUTING.rst]:
```suggestion

Ready to contribute?

In general, we ask you to follow the GitHub Flow workflow.

Here's how to set up simpler_eda for local development.
```

**Review by ChuckHo777 — COMMENTED** [review]
make change suggestion

**Pull request merged**